themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,32 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'div';
+  $wrapperClass = $this->wrapperClass ?? false;
+
+  $toggleTag = $this->toggleTag ?? 'button';
+  $toggleAttrs = $this->toggleAttrs ?? [];
+
+  $href = $this->href ?? $toggleTag == 'button' ? null : '#';
+?>
+
+<<?=$wrapperTag ?>
+  class="<?=$wrapperClass ? $this->escapeHtmlAttr($wrapperClass) . ' ' : '' ?>dropdown"
+>
+  <<?=$toggleTag ?>
+    class="dropdown-toggle"
+    data-toggle="dropdown"
+    <?php if ($href): ?>href="<?=$this->escapeHtmlAttr($href) ?>"<?php endif ?>
+  >
+    <?=$this->transEsc($this->toggleLabel) ?>
+    <?=$this->icon('dropdown-caret') ?>
+  </<?=$toggleTag ?>>
+
+  <ul class="dropdown-menu">
+    <?php foreach ($this->menuItems as $current): ?>
+      <li class="dropdown__item <?php if ($current['selected']): ?> active<?php endif ?>">
```

Comment:
Do we want to be tolerant of missing values in the menuItems array? For example, should we use `$current['selected'] ?? false` to assume that if the attribute is missing, it's not selected? I can see that url and label should probably be required, though...